extensions/amazon-aws/src/s3.tsx

```diff
@@ -70,8 +111,27 @@ function S3BucketObjects({ bucket, prefix = "" }: { bucket: Bucket; prefix?: str
               actions={
                 <ActionPanel>
                   <Action.Push
-                    target={<S3BucketObjects bucket={bucket} prefix={commonPrefix.Prefix} />}
+                    target={
+                      <S3BucketObjects
+                        bucket={bucket}
+                        prefix={commonPrefix.Prefix}
+                        isReversedOrder={isReversedOrder}
+                        setReversedOrder={setReversedOrder}
+                      />
+                    }
                     title="List Objects"
+                    icon={Icon.List}
+                  />
+                  <Action
+                    title={`${isReversedOrder ? "Standard" : "Reversed"} Order`}
+                    onAction={() => {
+                      setReversedOrder(!isReversedOrder);
+                      mutate();
```

Comment:
```suggestion
```

I don't think you need to `mutate` if we use cached promise correctly and pass the `isReversedOrder` as param.

Reply:
Seems we need to. The list won't be updated.

Reply:
It will work w/o requiring mutation (I tested it). The problem is that `setReverseOrder` is not working properly. Please see the other thread for how to fix it.

Reply:
Also we will need the `Reverse Order` action when working with `objects.objects` (list items) in addition to the prefixes block. Right now, you are only defining it for prefixes list items.